.github/workflows/deploy-tests.yml

```diff
@@ -29,7 +29,7 @@ jobs:
   run-tests:
     needs: assess-file-changes
     if: ${{ needs.assess-file-changes.outputs.SOURCE_CHANGED == 'true' }}
-    uses: catalystneuro/neuroconv/.github/workflows/testing.yml@main
+    uses: catalystneuro/neuroconv/.github/workflows/testing.yml@spikeglx_converter
```

Comment:
Is this part of getting around the streaming issues?

Reply:
We don't test streaming here in NeuroConv - the issues were with DANDI upload and Globus imports (both rely on `boto` at various levels)

But yes, relaxing the version on AWS CLI step of the testing workflows was a part of resolving the spontaneous issues on this PR

#420 should still happen though to avoid any kind of sudden live service interruption in the workflows going into the future

Reply:
can we revert this now?

Reply:
Done